### PR TITLE
Support all available lists via dropdown

### PR DESCRIPTION
**Pull request by lavolp3**
This PR supports showing all BRING lists via a dropdown.
List names are fetched through the getlists function and a dropdown gets filled with them. Every time a list is selected getlist() gets the contents.
PR is in beta stage, may need more testing and styling.
It works great so far for me.

image

Have a look at it and let me know what you think.

### CODE REVIEW

File: MMM-Bring.js [hunk 1]
```diff
@@ -19,43 +20,90 @@ Module.register("MMM-Bring", {
...
+        if (this.config.showListName && this.currentList && this.currentList.name) {
+
+            const dropTitle = this.createDropDown();
+            document.addEventListener("click", event => {
+                if (!event.target.matches('.bring-titleBtn')) {
+                    var dropDown = document.getElementById("bring-dropItems");
+                    if (dropDown.classList.contains('show')) {
+                        dropDown.classList.remove('show');
+                    }
+                    /*var dropdowns = document.getElementsByClassName("dropdown-content");
```
Comment (werthdavid) [inline MMM-Bring.js:74]: can be removed?

File: MMM-Bring.js [hunk 2]
```diff
@@ -129,9 +177,14 @@ Module.register("MMM-Bring", {
 
     socketNotificationReceived: function (notification, payload) {
         if (notification === "LIST_DATA") {
-            this.list = payload;
+            this.currentList = payload.currentList;
+            this.lists = payload.lists;
+            if (!this.config.listName) {
+                this.config.listName = this.currentList.name;
+            }
+            //console.log(JSON.stringify(this.lists));
```
Comment (werthdavid) [inline MMM-Bring.js:185]: unnecessary comment

File: MMM-Bring.js (hunk 1 above)
Comment (lavolp3) [reply MMM-Bring.js:74]: Oh sorry I accidentally resolved this. Will have a look into it.

Comment (lavolp3) [reply MMM-Bring.js]: Removed.

File: MMM-Bring.js (hunk 2 above)
Comment (lavolp3) [reply MMM-Bring.js]: Removed.

**Pull request merged**